modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -4,77 +4,203 @@ import {Meta} from '@storybook/addon-docs';
+
+**Do not upgrade to v8 if your application needs to support IE11.**
+
+---
+
+### Removal of Default Exports
+
+We've removed all remaining default exports from our packages to maintain consistency with our newer
+(and recently updated) components and systems which favor named exports. The following components
+have been updated to provide named exports instead of default exports (in cases where the component
+provided both default and named exports, the default export has been removed).
+
+#### Main (canvas-kit-react)
+
+- AccentIcon
+- AppletIcon
+- Avatar
+- Breadcrumbs (formerly in Preview)
+- Canvas
+- Checkbox
+- CountBadge
+- FormField
+- Graphic
+- Icon
+- LoadingDots
+- Radio
+- SegmentedControl
+- Select
+- SidePanel
+- StatusIndicator
+- Switch
+- Svg
+- SystemIcon
+- SystemIconCircle
+- Table
+- TextArea
+- TextInput
+- Toast
+- Tooltip
+
+#### Labs (canvas-kit-labs-react)
 
-**Breaking changes handled by this codemod will be marked with a 🤖.**
+- Combobox
+- Drawer
+
+#### Preview (canvas-kit-preview-react)
+
+- ColorPicker
+- Menu
+- Select
+- SidePanel
+
+#### Fonts (canvas-kit-react-fonts)
+
+- fonts
+
+🤖 The codemod will rewrite your default imports to named imports.
+
+---
+
+### Responsive Styles
+
+> Coming Soon (https://github.com/Workday/canvas-kit/pull/1779)
+
+---
+
+### Refactor Style Props
 
-**Please verify all changes made by the codemod. As a safety precaution, we recommend committing the
-changes from the codemod as a single isolated commit (separate from other changes) so you can
-rollback more easily if necessary.**
+> Coming Soon (https://github.com/Workday/canvas-kit/pull/1783)
 
-[Let us know](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you
-encounter any issues or use cases that we've missed.
+---
 
-## Components
+### Testing
 
-### Component Deprecations
+> Coming Soon (https://github.com/Workday/canvas-kit/pull/1800)
 
-#### Deprecation Types
+## New Components
 
-There are two types of deprecations: soft and hard.
+### Grid
 
-##### Soft Deprecation
+We've introduced a new `Grid` component to the Main package. `Grid` is a low-level layout component
+that provides a common, ergonomic API for building two-dimensional layouts (rows and columns) with
+[CSS Grid](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids).
+
+For more information, view the `Grid` component [docs](/components/layout/grid/).
+
+---
+
+### Text
+
+We've introduced a group of Text components to the Main package: `Text`, `LabelText`, `Title`,
+`Heading`, `BodyText`, and `Subtext`. Together they provide a common, ergonomic API for rendering
+text with built-in support for our Canvas Kit type tokens.
+
+For more information, view the Text component [docs](/components/text/).
+
+---
+
+### Status Indicator (Preview)
+
+> Coming Soon (https://github.com/Workday/canvas-kit/issues/1637)
+
+## Component Deprecations
```

Comment:
Note that I devoted an H2 for `## Component Deprecations` but not for Component Promotions. Promotions are instead highlighted under the respective component H3 under `## Component Updates` below. I felt that component deprecations warranted an extra call-out so I gave them their own H2 section. Component promotions felt like they worked within the context of `Component Updates`.

Let me know if you think differently.